cachi2/core/models/sbom.py

```diff
@@ -7,6 +7,7 @@
 PropertyName = Literal[
     "cachi2:bundler:package:binary",
     "cachi2:found_by",
+    "cachi2:rpm_summary",
```

Comment:
Should we use a custom cachi2 property or perhaps the description field for a CycloneDX Component instead?

Reply:
Maybe. I'll need to think about it more. A Property looks the least invasive way of doing this.

Reply:
FWIW properties can be straightforwardly converted to annotations.